mozilla-release/browser/base/content/browser-doctype.inc

```diff
@@ -2,6 +2,8 @@
 %brandDTD;
 <!ENTITY % browserDTD SYSTEM "chrome://browser/locale/browser.dtd" >
 %browserDTD;
+<!ENTITY % browserGhosteryDTD SYSTEM "chrome://browser/locale/browser.ghostery.dtd" >
```

Comment:
@aleksei-cliqz @alver-cliqz will this work easily in the future considering the updates?